Algorithm.Framework/Selection/BaseETFUniverseSelectionModel.cs

```diff
@@ -0,0 +1,84 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Data.UniverseSelection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuantConnect.Algorithm.Framework.Selection
+{
+    /// <summary>
+    /// Base ETF Universe that accepts a Dictionary of DateTime keyed by String that represent
+    /// the Inception date for each symbol
+    /// </summary>
+    public class BaseETFUniverseSelectionModel : FundamentalUniverseSelectionModel
```

Comment:
This can be improved with a new base universe selection model that does not load (coarse) fundamental data.